framework/src/controller/schema/constants.js

```diff
@@ -180,66 +296,78 @@ module.exports = {
 		type: 'object',
 		required: ['MIN', 'LIFETIME', 'KEYSGROUP'],
 		properties: {
-			MIN: {
-				$ref: 'min',
-			},
-			LIFETIME: {
-				$ref: 'lifetime',
-			},
 			KEYSGROUP: {
-				$ref: 'keysgroup',
-			},
-		},
-		additionalProperties: false,
-	},
-
-	minConstraints: {
-		id: 'min',
-		type: 'object',
-		required: ['MINIMUM', 'MAXIMUM'],
-		properties: {
-			MINIMUM: {
-				type: 'integer',
-				min: 1,
-			},
-			MAXIMUM: {
-				type: 'number',
-				format: 'keysgroupLimit',
-				min: 1,
-			},
-		},
-		additionalProperties: false,
-	},
-
-	lifetimeConstraints: {
-		id: 'lifetime',
-		type: 'object',
-		required: ['MINIMUM', 'MAXIMUM'],
-		properties: {
-			MINIMUM: {
-				type: 'integer',
-				min: 1,
-			},
-			MAXIMUM: {
-				type: 'integer',
-				min: 1,
+				type: 'object',
+				required: ['MIN_ITEMS', 'MAX_ITEMS'],
+				properties: {
+					MIN_ITEMS: {
+						type: 'integer',
+						min: 1,
+						default: 1,
+						description:
+							'Minimum allowed number of keys inside a Multisignature pool',
+					},
+					MAX_ITEMS: {
+						type: 'integer',
+						min: 1,
+						default: 15,
+						description:
+							'Maximum allowed number of keys inside a Multisignature pool',
+					},
+				},
+				default: {
+					MIN_ITEMS: 1,
+					MAX_ITEMS: 15,
+				},
+				additionalProperties: false,
 			},
-		},
-		additionalProperties: false,
-	},
-
-	keysgroupConstraints: {
-		id: 'keysgroup',
-		type: 'object',
-		required: ['MIN_ITEMS', 'MAX_ITEMS'],
-		properties: {
-			MIN_ITEMS: {
-				type: 'integer',
-				min: 1,
+			MIN: {
+				type: 'object',
+				required: ['MINIMUM', 'MAXIMUM'],
+				properties: {
+					MINIMUM: {
+						type: 'integer',
+						min: 1,
+						default: 1,
+						description:
+							'Minimum allowed number of signatures required to process a multisignature transaction',
+					},
+					MAXIMUM: {
+						type: 'number',
+						min: 1,
+						maximum: {
+							$data: '/MULTISIG_CONSTRAINTS/KEYSGROUP/MAX_ITEMS',
+						},
+						default: 15,
+						description:
+							'Maximum allowed number of signatures required to process a multisignature transaction',
+					},
+				},
 			},
-			MAX_ITEMS: {
-				type: 'integer',
-				min: 1,
+			LIFETIME: {
+				type: 'object',
+				required: ['MINIMUM', 'MAXIMUM'],
+				properties: {
+					MINIMUM: {
+						type: 'integer',
+						min: 1,
+						default: 1,
+						description:
+							'Minimum timeframe in which a multisignature transaction will exist in memory before the transaction is confirmed',
+					},
+					MAXIMUM: {
+						type: 'integer',
+						min: 1,
+						default: 72,
+						description:
+							'Maximum timeframe in which multisignature transaction will exist in memory before the transaction is confirmed',
+					},
+				},
+				default: {
+					MINIMUM: 1,
+					MAXIMUM: 72,
+				},
```

Comment:
Is this double `default` definition not redundant?
We declare `default` values for each property AND declare a `default` attribute?
Applies to multiple places in the `config` definitions.

Reply:
Yes it is redundant but its how `ajv` builtin solution works for `default` attribute. We can revisit it and have our own implementation for default attribute then this redundancy will be removed.